springfield/cms/templates/cms/free_form_page.html

```diff
@@ -10,12 +10,22 @@
 {% block content %}
 <div class="fl-page">
   <main class="fl-main">
+    <include:inline-notification
```

Comment:
Non-what's new pages should not display the update notification ;)

Reply:
Thanks @stephaniehobson ! I've been looking so much at the page with the notification that I didn't even noticed it there.